spec/device_detector/device_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'spec_helper'
+
+describe DeviceDetector::Device do
```

Comment:
Could you please provide at least one example for `nil` cases when calling `#name` and `#device_type`?